src/presets/deno.ts

```diff
@@ -12,11 +18,119 @@ export const deno = defineNitroPreset({
   },
   rollupConfig: {
     preserveEntrySignatures: false,
-    external: ["https://deno.land/std/http/server.ts"],
+    // external: ["https://deno.land/std/http/server.ts"],
     output: {
       entryFileNames: "index.ts",
       manualChunks: () => "index",
       format: "esm",
     },
   },
 });
+
+export const denoServer = defineNitroPreset({
+  extends: "node-server",
+  entry: "#internal/nitro/entries/deno-server",
+  commands: {
+    preview:
+      "deno run --unstable --allow-net --allow-read --allow-env ./server/index.mjs",
+  },
+  rollupConfig: {
+    output: {
+      hoistTransitiveImports: false,
+    },
+    plugins: [
+      inject({
+        modules: {
+          process: "process",
+          global: "global",
+          Buffer: ["buffer", "Buffer"],
+          setTimeout: ["timers", "setTimeout"],
+          clearTimeout: ["timers", "clearTimeout"],
+          setInterval: ["timers", "setInterval"],
+          clearInterval: ["timers", "clearInterval"],
+          setImmediate: ["timers", "setImmediate"],
+          clearImmediate: ["timers", "clearImmediate"],
```

Comment:
It will use "native" process, timers, buffer correct? (not browserify)

Reply:
yes, it generates (for example) in final bundle:

```js
import { setTimeout, setInterval } from "node:timers";
```